spec/features/admin/order_dashboard_spec.rb

```diff
@@ -0,0 +1,18 @@
+# require 'rails_helper'
```

Comment:
Wrote this test on this branch but ended up mostly refactoring. Will use for the actual branch for 33